packages/babel-helpers/README.md

```diff
@@ -19,3 +21,37 @@ const typeofHelper = helpers.get('typeof');
 t.isExpressionStatement(typeofHelper);
 // true
 ```
+
+Inside a plugin:
+
+```js
+export default {
+  visitor: {
+    UnaryExpression(path) {
+      // The .addHelper function adds, if needed, the helper to the file
+      // and returns an expression which references the helper
+      const typeofHelper = this.addHelper("typeof");
+      t.isExpression(typeofHelper); // true
+  }
+};
+```
+
+## How to define an helper?
+
+> **NOTE**: This package is only meant to be used by the packages inluded in this repository. There is no way for third-party plugins to define an helper.
+
+Helpers are defined in the `src/helpers.js` file.
+```js
+helpers.customHelper = defineHelper(`
+  import dep from "dependency";
+  const foo = 2;
+  export default function getFooTimesDepPlusX(x) {
+    return foo * dep() + x;
+  }
+`);
+```
+
+Helpers...
+ - ... must have a default export, which is the entry-point of the helper;
```

Comment:
Is it valid in english to "hoist" the first part of the sentence if it is common to all the items of the list? 😅

Reply:
It's valid but I wonder what it saves - rather than just repeating the one word?

Reply:
Agreed, I think repeating "Helpers" is fine here.